server/src/classes/fight.class.ts

```diff
@@ -100,6 +100,20 @@ export class FightImpl
     else return undefined;
   }
 
+  getConnectedJudgesStatus(): Map<JudgeRole, JudgeState> {
+    const statusMap = new Map<JudgeRole, JudgeState>();
+    if (this.mainJudge.socket != null) {
+      statusMap.set(JudgeRole.MainJudge, this.mainJudge);
+    }
+    if ((this, this.redJudge.socket != null)) {
```

Comment:
O co chodzi z ((this, ...)) ? Wydaje mi się, że tu mogła być mała literówka i linter poprawił to na jakąś dziwną konstrukcję. Jeśli to nie było zamierzone, trzeba będzie poprawić ten warunek, bo to zaciemnia kod i może dziwnie działać.

Reply:
rzeczywiście jest podejrzane i ręcznie tego nie napisałem